src/twosat.rs

```diff
@@ -1 +1,107 @@
+use crate::internal_scc;
 
+pub struct TwoSat {
+    n: usize,
+    scc: internal_scc::SccGraph,
+    answer: Vec<bool>,
+}
```

Comment:
Maybe you can add `#[derive(Default)]`, because the original library provides default constructor, although I doubt it is useful.

Reply:
The original ACL's default constructor is for global variables.
For example, problems like [this](https://atcoder.jp/contests/arc069/tasks/arc069_d) requires traversing on a tree, adding constraints in a traversal like the following:
```
two_sat ts;

void dfs() {
  /* Add constraints to ts */
  :
  :
}

int main() {
  int n;
  cin >> n;
  ts = two_sat(n);
  :
  :
  dfs()
}
```

Having default constructors helps in such a situation in *C++*, but I don't think it also helps in Rust.

Reply:
I see.  It's like I have done a useless review...

Reply:
OK.
I will revert the applicable commit.(209a624)

Reply:
Sorry to bother you.